mappings/net/minecraft/world/MobSpawnerEntry.mapping

```diff
@@ -1,10 +1,10 @@
 CLASS net/minecraft/class_1952 net/minecraft/world/MobSpawnerEntry
 	FIELD field_30977 DEFAULT_ENTITY_ID Ljava/lang/String;
-	FIELD field_9307 entityTag Lnet/minecraft/class_2487;
+	FIELD field_9307 entityNbt Lnet/minecraft/class_2487;
 	METHOD <init> (ILnet/minecraft/class_2487;)V
 		ARG 1 weight
-		ARG 2 entityTag
+		ARG 2 entityNbt
 	METHOD <init> (Lnet/minecraft/class_2487;)V
-		ARG 1 tag
+		ARG 1 nbt
 	METHOD method_8678 getEntityNbt ()Lnet/minecraft/class_2487;
 	METHOD method_8679 serialize ()Lnet/minecraft/class_2487;
```

Comment:
```suggestion
	METHOD method_8679 toNbt ()Lnet/minecraft/class_2487;
```
For this class, I don't see why we cannot just call this field `entity` (has `Entity` key in compound representation)

Reply:
I still would keep an entity in the NBT form as `entityNbt` to avoid possible confusion maybe?
I'm not sure.

Reply:
Sure, that doesn't hurt at all :+1: